docs/configuring-playbook-dimension.md

```diff
@@ -46,6 +47,17 @@ To get an access token for the Dimension user, follow these steps:
 3. Copy the highlighted text to your configuration.
 4. Close the private browsing session. **Do not log out**. Logging out will invalidate the token, making it not work.
 
+*With CURL* 
+
+```
+curl -X POST --header 'Content-Type: application/json' -d '{
+    "identifier": { "type": "m.id.user", "user": "YourDimensionUsername" },
+    "password": "YourDimensionPassword",
+    "type": "m.login.password"
+}' 'http://localhost:8008/_matrix/client/r0/login'
```

Comment:
Most people run with `matrix_nginx_proxy_enabled: true` (the default). In that case, Synapse is not exposed to `localhost:8008`. We'd better change this to `https://matrix.DOMAIN`

Reply:
Good point, made the change.